src/AssemblerAlign.cpp

```diff
@@ -527,7 +527,7 @@ void Assembler::accessCompressedAlignments()
 void Assembler::computeAlignmentTable()
 {
     alignmentTable.createNew(largeDataName("AlignmentTable"), largeDataPageSize);
-    alignmentTable.beginPass1(ReadId(2 * reads.readCount()));
+    alignmentTable.beginPass1(ReadId(2 * reads->readCount()));
```

Comment:
At another time, we should perhaps add a member function `Assembler::readCount()`. But this is for another day.